flink-connectors/flink-connector-kafka-base/src/test/java/org/apache/flink/streaming/connectors/kafka/KafkaTestBase.java

```diff
@@ -118,8 +117,6 @@ public static void shutDownServices() throws Exception {
 
 	protected static Configuration getFlinkConfiguration() {
 		Configuration flinkConfig = new Configuration();
-		flinkConfig.setString(AkkaOptions.WATCH_HEARTBEAT_PAUSE, "5 s");
```

Comment:
shouldn't we update these instead to use the new heartbeat options?

Reply:
The test is using an unused configuration option for several releases. Hence, it seems to work with the default of the heartbeat options for quite some time. Hence, I would not touch if there is not a very good reason for it.